.github/workflows/code.yml

```diff
@@ -48,61 +48,78 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        language: [ 'csharp' ]
+        language:
+          - csharp
+          - javascript
+          - typescript
```

Comment:
Should be `javascript-typescript`

Reply:
I saw that comments, but i also checked the official documents, it lists javascript and typescript seperately. So i think it should be fine. At least it works well here.